Views/BuildingBlocks/WebView.swift

```diff
@@ -130,15 +130,11 @@ final class WebViewController: UIViewController {
 // MARK: - UIScrollViewDelegate
 extension WebViewController: UIScrollViewDelegate {
     func scrollViewDidScroll(_ scrollView: UIScrollView) {
```

Comment:
This whole block of this file is already under ``#if os(iOS)`` so we are good in terms of macOS.